library/src/main/java/com/novoda/downloadmanager/DownloadMigrationService.java

```diff
@@ -0,0 +1,12 @@
+package com.novoda.downloadmanager;
+
+interface DownloadMigrationService {
```

Comment:
Interface that we expose rather than the `android` version of `Service`.

Reply:
This is only used by the library. The clients interact with the `DownloadMigrator` (not an interface).

Reply:
Is there any need for it if it's not gonna be public? We already know that `LiteDownloadMigrationService` implements this. Every time it's used we downcast a `binder.getService()` result to LiteDownloadMigrationService` and then upcast that to `DownloadManagerService`. Feels a bit pointless. What do you think @zegnus?